reddit_liveupdate/public/static/css/liveupdate-public.less

```diff
@@ -0,0 +1,123 @@
+// CSS in here is imported into reddit's main CSS file, for inclusion of live features into reddit proper
```

Comment:
Hm - I was thinking through that option above this morning and my intuition is that it will be pretty hairy, specifically when thinking through both dev and production asset building. The generated file would need to be stored somewhere that the import would have uniform relative access to, and I don't know if I want to be shoving generated assets into `public/static/css`.

Any ideas appreciated. In the interest of expediency I'm considering adding these rules to the main reddit.less file, especially considering they'll have no effect if the plugin is not installed. I don't love that though, as it's pretty unexpected for folks doing liveupdate development down the line.